doc/whats_new/v1.1.rst

```diff
@@ -206,6 +206,11 @@ Changelog
   :class:`ensemble.ExtraTreesClassifier`.
   :pr:`20803` by :user:`Brian Sun <bsun94>`.
 
+- |Fix| solve a bug in :class:`ensemble.GradientBoostingClassifier` where the
+  the exponential loss was computing the positive gradient instead of the
```

Comment:
```suggestion
- |Fix| Solve a bug in :class:`ensemble.GradientBoostingClassifier` where the
  exponential loss was computing the positive gradient instead of the
```